dice_ml/explainer_interfaces/dice_genetic.py

```diff
@@ -260,7 +264,7 @@ def _generate_counterfactuals(self, query_instance, total_CFs, initialization="k
                  (see diverse_counterfactuals.py).
         """
 
-        self.population_size = 10 * total_CFs
```

Comment:
Why is this changed from 10 to 3?

Reply:
IMO this is quite indifferent with a small dataset and increases too much the computation time! However, if you want, I'll let it to 10!

Reply:
Now, is it ok? @gaugup